ide/Espressif/ESP-IDF/component-manager/wolfssl_component_publish.sh

```diff
@@ -0,0 +1,818 @@
+#!/bin/bash
```

Comment:
This script is touching things in the wolfMQTT directory, wolfSSH doesn't have that.

Reply:
It is a parameterized, universal script: the same for wolfSSL, wolfSSH, and wolfMQTT.

Perhaps an argument for a repo of it's own.